src/test/java/com/wikia/webdriver/testcases/discussions/ReportingPostTests.java

```diff
@@ -705,10 +701,10 @@ private void moderatorCanDeleteReportedPostOnPostDetailsPage() {
     final PostEntity.Data data = createAndReportPostRemotelyAsFirstUser();
     final PostDetailsPage page = openPostDetailsPageAndWaitUntilLoaded(data.getId());
     final PostEntity postEntity = page.getPost().findPostById(data.getId());
-    assertTrue(isReported(postEntity), REPORTED_INDICATOR_ON_POST_MESSAGE);
-    assertTrue(clickCancelOnDeletePostModalDialog(page, postEntity),
+    Assertion.assertTrue(isReported(postEntity), REPORTED_INDICATOR_ON_POST_MESSAGE);
+    Assertion.assertTrue(clickCancelOnDeletePostModalDialog(page, postEntity),
             REPORTED_INDICATOR_ON_POST_MESSAGE);
-    assertTrue(clickApproveOnDeletePostModalDialog(page, postEntity), DELETED_POST_MESSAGE);
+    Assertion.assertTrue(clickApproveOnDeletePostModalDialog(page, postEntity), DELETED_POST_MESSAGE);
   }
 
   private UserPostsPage openUserPostsAndWaitUntilLoaded(String authorId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openUserPostsAndWaitUntilLoaded" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)